openedx/features/enterprise_support/signals.py

```diff
@@ -105,8 +106,14 @@ def refund_order_voucher(sender, course_enrollment=None, skip_refund=False, **kw
     client = ecommerce_api_client(service_user)
     order_number = course_enrollment.get_order_attribute_value('order_number')
```

Comment:
Is course_enrollment ensured to be not None? If not, we should verify before calling a method.

Reply:
I could find only one [location](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/models.py#L1348) where are we broadcasting this signal.
It can't be None here.